src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FindT.txt

```diff
@@ -128,29 +128,24 @@ Table({Result:3},{Result:Microsoft.PowerFx.Core.Public.Values.ErrorValue},{Resul
 Table({Result:3},{Result:Microsoft.PowerFx.Core.Public.Values.ErrorValue},{Result:1})
 
 >> Find("c", ["abc", Error({Kind: "Validation"}), "cde"])
```

Comment:
I know this is not part of your changes, but since you're changing this file, can you replace all instances of `Error({Kind: "Validation"})` with `Error({Kind:ErrorKind.Validation})`? They are not useful today: they don't work in the JS interpreter, and it's only "working" in the C# interpreter because the Error function is not implemented (so it's currently returning an error). Similar for the new 'FindT.txt' file in the new folder.